gha/checkout/action.yaml

```
@@ -23,15 +23,17 @@ runs:
   using: "composite"
   steps:
     - name: Configure git user
+      shell: bash
       env:
         GIT_USER_NAME: "${{ inputs.git_user_name }}"
         GIT_USER_EMAIL: "${{ inputs.git_user_email }}"
       run: |
-        git config --global user.name "${GIT_USER_NAME}"
-        git config --global user.email "${GIT_USER_EMAIL}"
+        git config --global user.name "${{env.GIT_USER_NAME}}"
+        git config --global user.email "${{env.GIT_USER_EMAIL}}"
```

We can use `env` to check out the env and make sure our env vars are available.
```suggestion
        env
        git config --global user.name "${GIT_USER_NAME}"
        git config --global user.email "${GIT_USER_EMAIL}"
```